meson.build

```diff
@@ -2,6 +2,7 @@ project('flexalloc', 'c', default_options : ['c_std=c11'])
 cc = meson.get_compiler('c')
 pymod = import('python')
 py = pymod.find_installation('python3', modules: ['pip', 'venv'], required: true)
+pconf = import('pkgconfig')
```

Comment:
Do we use it any other place than in fio-ioengine/meson.build? Should we just importe it there?

Reply:
True, shouldn't be added here, yet. Though we should generate a pkg-config file for libflexalloc (different PR).

Reply:
removed in new commit.